sdkcompat/v242/com/google/idea/sdkcompat/clion/CMakeNotificationProviderWrapper.java

```diff
@@ -17,32 +17,20 @@
 package com.google.idea.sdkcompat.clion;
 
 
+import com.intellij.openapi.extensions.ExtensionPoint;
 import com.intellij.openapi.fileEditor.FileEditor;
 import com.intellij.openapi.project.Project;
 import com.intellij.openapi.vfs.VirtualFile;
-import com.jetbrains.cidr.cpp.cmake.workspace.CMakeNotificationProvider;
-
-import java.util.function.Function;
 import javax.annotation.Nullable;
-import javax.swing.*;
+import javax.swing.JComponent;
 
-// #api223
+// #api224 - CMakeNotificationProvider was removed in ths release
```

Comment:
So the markers are meant to reference "the last release where this piece of code is required". With this notation, we can easily grep for "#api223" when we will be deleting 223 support